src/scenes/CreateEvent/index.js

```diff
@@ -260,6 +269,13 @@ class CreateEvent extends React.Component {
         <Form onSubmit={handleSubmit(this.submitCreateEvent)}>
           <DialogTitle>{intl.formatMessage(messages.dialogTitle)}</DialogTitle>
           <DialogContent>
+            <div className={classes.importantNoteContainer}>
+              <ImportantNote
+                className={classes.createEscrowNote}
+                heading={intl.formatMessage(messages.escrowNoteTitle)}
```

Comment:
use `eventEscrowAmount` prop to get the dynamic amount and use placeholder string